mmv1/third_party/terraform/tests/resource_compute_router_bgp_peer_test.go

```diff
@@ -178,6 +178,126 @@ func TestAccComputeRouterPeer_routerApplianceInstance(t *testing.T) {
 	})
 }
 
+func TestAccComputeRouterPeer_Ipv6Basic(t *testing.T) {
+	t.Parallel()
+
+	routerName := fmt.Sprintf("tf-test-router-%s", RandString(t, 10))
+	resourceName := "google_compute_router_peer.foobar"
+	VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { testAccPreCheck(t) },
```

Comment:
```suggestion
		PreCheck:                 func() { AccTestPreCheck(t) },
```

Reply:
@shuyama1 Thanks for pointing this out. I'll fix the naming